CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add `GetFolderContent`: fetches the contents of a folder.
```

Comment:
```suggestion
- Add `GetFolderContent` which fetches the contents of a folder.
```

Reply:
I was just following the format of the file. But I do prefer it this way too so I'll merge it :)